docs/upgrading-to-v0.18.x.md

```diff
@@ -0,0 +1,244 @@
+# Upgrading to v0.18.x
+
+Starting with `v0.18.0` version, [provider-gcp] will use `v1` version of GCP
+APIs for container group instead of `v1beta1`. Please note that, this is not
```

Comment:
Might be worth mentioning that this is true for all types, it just happened to affect the container resources in this change.

Reply:
Done